security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersDeserializer.java

```diff
@@ -41,12 +41,25 @@ public SecurityAnalysisParameters deserialize(JsonParser parser, Deserialization
                 case "version":
                     parser.nextToken();
                     break;
-
+                case "increasedFlowViolationsThreshold":
+                    parameters.setIncreasedFlowViolationsThreshold(Double.parseDouble(parser.nextTextValue()));
```

Comment:
See comment in JSON file: should be parsed straight as double, not as text and then parsed.

Reply:
Done